Library/Formula/libvisio.rb

```diff
@@ -15,10 +14,11 @@ class Libvisio < Formula
   depends_on "pkg-config" => :build
   depends_on "boost" => :build
   depends_on "cppunit" => :build
-  depends_on "libwpd"
-  depends_on "libwpg"
-  depends_on "icu4c"
   depends_on "librevenge"
+  depends_on "icu4c"
+  depends_on "libxml2"
+  depends_on "homebrew/dupes/gperf"
+  depends_on "homebrew/dupes/zlib"
```

Comment:
ok.. so libvisio can be compiled without homebrew's libxml2, gperf, libz 
the icu4c is not a new dep. It's in the current homebrew formula. 
I dont know how to squash commits well.. can you just squash them and commit ?